shared/teams/role-button/index.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import {InlineDropdown} from '../../common-adapters/dropdown'
+import {TeamRoleType} from '../../constants/types/teams'
+import {roleIconMap} from '../role-picker'
+import capitalize from 'lodash/capitalize'
+
+export type Props = {
+  selectedRole: TeamRoleType
+  onClick: () => void
+}
+
+const RoleButton = (props: Props) => {
+  const iconType = roleIconMap[props.selectedRole]
+
+  return (
+    <InlineDropdown
+      type="BodySmallSemibold"
+      label={
+        <Kb.Box2 direction="horizontal" alignItems="center" style={styles.label}>
+          <Kb.Icon type={iconType} style={styles.icon} fontSize={Styles.isMobile ? 16 : 12} />
```

Comment:
I think this fontSize business is equivalent to `sizeType="Default"` which is preferable.

Reply:
It's in fact `Small`, I forgot about `sizeType`, cheers!